test/Tests/Readers/Markdown.hs

```diff
@@ -307,6 +307,26 @@ tests = [ testGroup "inline code"
             "[https://example.org(](url)" =?>
             para (link "url" "" (text "https://example.org("))
           ]
+        , testGroup "Github wiki links"
+          [ test markdownGH "autolink" $
+            "[[https://example.org]]" =?>
+            para (link "https://example.org" "wikilink" (text "https://example.org"))
+          , test markdownGH "autolink not being a link" $
+            "[[Name of page]]" =?>
+            para (link "Name of page" "wikilink" (text "Name of page"))
+          , test markdownGH "autolink not being a link with a square bracket" $
+            "[[Name of ]page]]" =?>
+            para (link "Name of ]page" "wikilink" (text "Name of ]page"))
+          , test markdownGH "autolink not being a link with strong" $
+            "[[Name of *page*]]" =?>
+            para (link "Name of *page*" "wikilink" (text "Name of " <> emph "page"))
+          , test markdownGH "link with title" $
+            "[[title|https://example.org]]" =?>
+            para (link "https://example.org" "wikilink" (text "title"))
+          , test markdownGH "bad link with title" $
+            "[[title|random string]]" =?>
+            para (link "random string" "wikilink" (text "title"))
+          ]
         , testGroup "Headers"
```

Comment:
Does GitHub (or the markdown parser they use in wikis, which I think may be kramdown) have a suite of tests for this syntax?  If so, it may be worth finding it and seeing if any relevant tests can be added.